docs/ADOPTING_DARIA.md

```diff
@@ -16,7 +21,12 @@ Every fund's DARIA instance is separate from others. Your DARIA instance and its
 
 ## Security
 
-[LISA FILLS IN]
+By reaching out to a fund, a patient is implicitly trusting us to be discreet and safeguard their personal data. To make good on funds' promise to keep our patients safe and offer them the best advice and support we can, our development team's top priority is keeping patient data secure and safe through deploying industry best practices and security monitoring softwares. 
+
+Additionally, we know any tech is only as secure as the people using it. A fund should also maintain secure data collection and usage practices at the individual case manager level, including deploying two factor authentication on their google accounts and using DARIA's features rather than using potentially compromising patient information in email communications.
+
+Find our complete security review at:
+https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SECURITY.md
 
 ## How to migrate
 
```

Comment:
Migration is a manual process using an interface developed to streamline data entry (https://cmapp.dcabortionfund.org/data_entry). It's best to set up a team for this process, especially if there are a large number of patients to migrate. Each of the volunteers doing data entry will need a login in for DARIA.

Instructions to use the expedited data entry form: 

* Log in as normal
* If multiple lines, select the assigned line 
* Go to https://cmapp.dcabortionfund.org/data_entry
* Enter the patient information from the spreadsheet, except for notes and external pledges, and hit submit
* Manually add the notes and external pledges
* Navigate back to https://cmapp.dcabortionfund.org/data_entry 
* Repeat until complete

Once you've created the patient, go into 'Notes' tab.  Copy text from "Comments & Patient Story" to the text box.  
Once you've finished with notes, go into the 'Call Log' tab.  Click on the 'Record new call' link.  Select option that matches 'Status' cell in spreadsheet.

Some tips: 
Sometime the notes in Crypto contain information that will have a separate field in DARIA -- read the notes.  
For longer notes in Crypto, use the date stamps (e.g. // 12/2) to break up into smaller notes for DARIA.